src/Pages/Repositories/ContentListTable/components/DeleteContentModal/DeleteContentModal.tsx

```diff
@@ -102,7 +106,11 @@ export default function DeleteContentModal() {
     sortString,
   );
 
-  const onClose = () => navigate(`${rootPath}/${REPOSITORIES_ROUTE}`);
+  const onClose = () =>
+    navigate(
+      `${rootPath}/${REPOSITORIES_ROUTE}${checkedPopularRepos ? `/${POPULAR_REPOSITORIES_ROUTE}` : ''}`,
```

Comment:
looks like this will navigate to the popular repos page on close even if checkedPopularRepos is empty. maybe we should check`checkedPopularRepos.length != 0` instead?